backend/src/routes/recipes.js

```diff
@@ -4,9 +4,27 @@ const router = express.Router();
 const queries = require("../prisma/recipes");
 
 /* GET recipes listing. */
+// todo
 router.get('/', async function (req, res, next) {
   const result = await queries.getRecipes();
   res.json(result)
 });
 
+/* DELETE recipe. */
+router.delete('/:id', async function (req, res, next) {
+  try {
+      await queries.deleteRecipe(req.params.id);
+      res.status(204)
+  }
+  catch (e) {
+      if (e instanceof DeleteIngredientError)  {
```

Comment:
This error cannot be caught when it is not thrown in deleteRecipe in recipes.js.
This error is for Ingredient. We need new DeleteRecipeError in https://github.com/benkeanna/MixBook/blob/main/backend/src/errors.js.

Reply:
smaze se error v dalsim commentu